src/elements/Button/Button.styl

```diff
@@ -123,17 +138,16 @@
   transition-duration $duration-shortest
 
   // Children
-
   & > .content
-    padding $space-normal
-
+    padding $space-zero
+    padding-bottom 7px
     &.has-addons
       border-left 1px solid var(--button-border-color)
-
+      
```

Comment:
Trailing spaces must go. This applies to all blank lines in all files.

Reply:
```suggestion
      & > .addons
         &.padded
            padding $space-normal	
  // States	  // States
```